akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -677,22 +677,19 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
 
   private def ormapDeltaGroupToProto(deltaGroup: ORMap.DeltaGroup[_, _]): rd.ORMapDeltaGroup = {
```

Comment:
it's scheduled to not run during the night because we have many other jobs running then